From 99bdb7726733ae4e797ee3bfbd3a19c7227a5e67 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Sun, 14 Jul 2024 11:49:39 -0600
Subject: [PATCH] Bump Security String to 2024-07-05

Implemented:
============
CVE:            References:    Type:  Severity:  Updated AOSP versions:
CVE-2024-34720  A-319081336    EoP    High       12, 12L, 13, 14
CVE-2024-34721  A-294406604    ID     High       12, 12L, 13, 14
CVE-2024-34722  A-251514170    EoP    High       12, 12L, 13, 14

Not Implemented:
================
None

Not Applicable (platform source):
=================================
CVE:            References:    Type:  Severity:  Updated AOSP versions:
CVE-2024-31320  A-329230490    EoP    Critical   12, 12L
CVE-2024-31331  A-297517712    EoP    High       12, 12L, 13, 14
CVE-2024-31332  A-299931076    EoP    High       13, 14
CVE-2024-31339  A-292160348    EoP    High       12, 12L, 13, 14
CVE-2024-34723  A-317048338    EoP    High       12, 12L, 13, 14

Change-Id: Icf1d280db7da32e7406c460cb513ec201e20f702
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 2ca5825664..58b3d1294a 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -250,7 +250,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-    PLATFORM_SECURITY_PATCH := 2024-06-05
+    PLATFORM_SECURITY_PATCH := 2024-07-05
 endif
 .KATI_READONLY := PLATFORM_SECURITY_PATCH
 
-- 
2.40.1

